main.js

```diff
@@ -156,11 +189,12 @@ function retrieveNewVideo() {
   if (document.title == "Secret~") history.pushState({video: "Egg", list: []}, document.title, location.origin + location.pathname);
   else history.pushState({video: vNum, list: video_obj}, document.title, location.origin + location.pathname);
 
-  ++vNum;
-
   resetSubtitles();
   document.getElementById("bgvid").play();
-  $("#pause-button").toggleClass("fa-play").toggleClass("fa-pause");
+  document.getElementById("pause-button").classList.remove("fa-play");
+  document.getElementById("pause-button").classList.add("fa-pause");

```

Comment:
I'd prefer it if this were
`$("#pause-button").removeClass("fa-play").addClass("fa-pause");`
(It's shorter and more idiomatic)